src/main/java/org/broadinstitute/hellbender/utils/read/BDGAlignmentRecordToGATKReadAdapter.java

```diff
@@ -0,0 +1,40 @@
+package org.broadinstitute.hellbender.utils.read;
+
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.SAMRecord;
+import org.bdgenomics.adam.converters.AlignmentRecordConverter;
+import org.bdgenomics.adam.models.SAMFileHeaderWritable;
+import org.bdgenomics.formats.avro.AlignmentRecord;
+
+import java.util.UUID;
+
+/**
+ * Implementation of the {@link GATKRead} interface for the {@link AlignmentRecord} class.
+ *
+ * This adapter wraps an {@link AlignmentRecord} by first converting to a {@link SAMRecord} and then passing through
+ * to {@link SAMRecordToGATKReadAdapter}.
+ */
+public final class BDGAlignmentRecordToGATKReadAdapter extends SAMRecordToGATKReadAdapter {
+    private static final long serialVersionUID = 1L;
+
+    private final AlignmentRecord alignmentRecord;
+
+    public BDGAlignmentRecordToGATKReadAdapter( final AlignmentRecord alignmentRecord) {
+        this(alignmentRecord, null);
+    }
+
+    public BDGAlignmentRecordToGATKReadAdapter( final AlignmentRecord alignmentRecord, final SAMFileHeaderWritable header) {
```

Comment:
Actually, I'm gonna change it from `SAMFileHeaderWritable` to `SAMFileHeader`.  The `Writable` is an ADAM class, and the regular htsjdk one is already serializable.